minimizer/minimize.sh

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash
+
```

Comment:
```suggestion
if [ $# -lt 2 ]; then
    echo "usage: $0 <file> <condition>"
    echo "note: see 'minimizer/conditions' for valid conditions"
    exit 1
fi

```

It would also be nice to move this check later so it can point to `$SCRIPT_PATH/conditions` instead, but that's a little to much to do from the github editor :laughing:

Reply:
I changed a bit more and also the ordering. do you mind checking this and resolving this if its okay?

Reply:
Looks great :) I love how it prints the available conditions.